Text input skill

asorichetti opened the pull request:
The work that is done in getting user input text, and validating that text based on set parameters for answers that I am looking for.

asorichetti committed:
Created text input "form" function, and placed
within profile

asorichetti committed:
added basic input validation causing change based
on user input into placed textbox

asorichetti committed:
added basic input validation

asorichetti committed:
Made it so rather than reloading the page
you can simply delete what was typed in the box
and change the answer.

asorichetti committed:
End of day with data verification system.

hweeks commented on src/components/Profile/index.tsx:
```diff
@@ -15,18 +17,52 @@ const qualifications = [
+    function handleChange(event: React.ChangeEvent<HTMLInputElement>){
+        setValue(event.currentTarget.value);
+    }
+    if (value == 'yes' || value == 'Yes'){
```
this would be better as:

```js
if (value.toLowerCase() === "yes")
```

as it captures all possible casing

hweeks commented on src/components/Profile/index.tsx:
```diff
@@ -15,18 +17,52 @@ const qualifications = [
+            <input value={value} onChange={handleChange} />
+             <p>Would You Hire Me?: {value}</p>
+             </div>
+        )
```
ok i see, so fo this you wanna be more like:

```tsx
let myResponse = <></>
if (value.toLowerCase() === "yes") {
  myResponse = <div>hell yea email me</div>
} else {
  myResponse = <div>everyone is a critic</div>
}

return (
  <>
     {myResponse}
     <input value={value} onChange={handleChange} />
     <p>Would You Hire Me?: {value}</p>
  <>
)
```

hweeks commented on src/components/Profile/index.tsx:
```diff
@@ -47,7 +83,10 @@ export function Profile() {
                 }
                 )}
-                <MyButton />
+                <p></p>
```
you can use a `<br />` for this instead

asorichetti committed:
updated code to reflect any capitalization as well
as having a default case of "learning is always
happening" that will always appear unless the user
types in "yes" or "no"

asorichetti committed:
Just minor changes to formatting with a change to
<p/> rather than double sets of <p> and </p>

asorichetti committed:
Added fonts
